site/docs/components/combo-box/accessibility.mdx

```diff
@@ -13,8 +13,8 @@ data:
 <KeyboardControls>
 <KeyboardControl keyOrCombos="Tab">
   
-- If the combo box is single-select and if focus is above the combo box, Tab sets focus on the component. When focused, combo box has a visible text cursor and the list is opened.
-- If the combo box is multi-select and if the focus is above the combo box, Tab sets focus on the pill list if pills are present. If no pills are present then focus is set on the input. 
+- If the combo box is single-select and if focus is above the combo box, Tab sets focus on the component. When focused, combo box has a visible text cursor and the list is opened. After typing a value, if tab is pressed, by default the value gets selected from the list. Pass 'tabToSelect' prop as false to not select a typed value on tab press.
```

Comment:
This guidance needs to be updated. Before the sentences we small and easy to follow but we're added a couple of extra sentences to both which makes it harder to follow.

Reply:
Hi @joshwooding , I was just reading the previous version of this, and the behaviour is actually same. Do I actually need to add something here for the prop added?